src/App.tsx

```diff
@@ -8,13 +12,18 @@ export const App = () => {
     <div data-cy="app">
       <Navbar />
 
-      <div className="section">
+      <main className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/home" element={<Navigate to="/" replace={true} />} />
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path=":slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested route for `PeoplePage` might not work as expected because the parent route `<Route path='/people' element={<PeoplePage />}>` does not have an `Outlet` component to render the nested route. Consider using an `Outlet` in `PeoplePage` to properly render nested routes.